tool/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@
 add_executable(foonathan_memory_node_size_debugger test_types.hpp node_size_debugger.hpp node_size_debugger.cpp)
 if (CMAKE_CROSSCOMPILING)
     # statically link when cross compiling so emulator doesn't need library paths
-    set_target_properties(foonathan_memory_node_size_debugger PROPERTIES LINK_FLAGS "-static")
+    set_target_properties(foonathan_memory_node_size_debugger PROPERTIES LINK_FLAGS "/WHOLEARCHIVE")
```

Comment:
This breaks cross-compiling in Linux

Reply:
Should be fixed by the commit I've referenced below, please check and open an issue if it doesn't work.